lib/kebab_van.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'take_away'
+
+class KebabVan < TakeAway
+  attr_accessor :menu, :time, :name
+
+  def initialize(name)
+    @menu = {'Burger' => 2, 'Doner-Kebab' => 4, 'Chips-and-Cheese' => 3, 'Chicken-Kebab' => 5, 'Mixed-Kebab' => 4, 'Chicken-Burger' =>3}
```

Comment:
Line is too long. [136/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.